extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexus.js

```diff
@@ -65,8 +65,8 @@ define('ext.wikia.adEngine.lookup.prebid.adapters.appnexus',[
 		return adContext.get('bidders.appnexus');
```

Comment:
Does it mean that AN is going to be enabled both for BT and IL rec? Shouldn't we enable it based on `context.opts.wadIL`?

Reply:
That's true, condition expanded.